src/main/java/fr/skytasul/guardianbeam/Laser.java

```diff
@@ -327,18 +327,18 @@ private void initLaser() throws ReflectiveOperationException {
 			teamCreatePacket = Packets.createPacketTeamCreate("noclip" + teamID.getAndIncrement(), squidUUID, guardianUUID);
 			destroyPackets = Packets.createPacketsRemoveEntities(squidID, guardianID);
 		}
-		
-		private void initSquid() throws ReflectiveOperationException {
-			if (Packets.version < 17) {
-				squid = null;
-				createSquidPacket = Packets.createPacketEntitySpawnLiving(end, Packets.mappings.getSquidID(), squidUUID, squidID);
-			}else {
-				squid = Packets.createSquid(end, squidUUID, squidID);
-				createSquidPacket = Packets.createPacketEntitySpawnLiving(squid);
-			}
-			metadataPacketSquid = Packets.createPacketMetadata(squidID, Packets.fakeSquidWatcher);
-			Packets.setDirtyWatcher(Packets.fakeSquidWatcher);
-		}
+    
+    	private void initSquid() throws ReflectiveOperationException {
+        	if (Packets.version < 17) {
+        	    squid = null;
+        	    createSquidPacket = Packets.createPacketEntitySpawnLiving(end, Packets.mappings.getSquidID(), squidUUID, squidID);
+        	} else {
+        	    squid = Packets.createSquid(end, squidUUID, squidID);
+        	    createSquidPacket = Packets.createPacketEntitySpawnLiving(squid);
+        	}
+        	metadataPacketSquid = Packets.createPacketMetadata(squidID, Packets.fakeSquidWatcher);
+        	Packets.setDirtyWatcher(Packets.fakeSquidWatcher);
+    	}
```

Comment:
You should not reformat existing code.

Reply:
Not sure how that happened, most likely from resolving conflicts after 2.2.0